### DISCUSSION TITLE
泳道的引入

### DISCUSSION THREAD

**Discussion by MarthaRen** [didi/LogicFlow#1289] [Ideas]
请问团队后续会考虑bpmn 泳道等模型的引入吗

**Comment by wumail** [comment]
![image](https://github.com/didi/LogicFlow/assets/56008486/534f5439-d4c9-4f02-a440-8cfd8ea30933)
https://docs.logic-flow.cn/examples/#/gallery 上有简单的泳道实现，可以参考一下。本质上是group节点嵌套的自定义节点，后续会优化、完善并集成到bpmn-elements插件中

**Comment by MarthaRen** [comment]
ok
